setup.py

```diff
@@ -34,14 +35,11 @@
 # this sets __version__
 # via: http://stackoverflow.com/a/7071358/87207
 # and: http://stackoverflow.com/a/2073599/87207
-with open(os.path.join("capa", "version.py"), "r") as f:
-    exec(f.read())
+exec(Path("capa/version.py").read_text())
```

Comment:
```suggestion
exec((Path("capa") / "version.py").read_text())
```

we want to avoid hardcoding path separators